python/test/unit/language/test_warp_specialization.py

```diff
@@ -343,10 +343,11 @@ def matmul_tma_persistent_ws_kernel(  #
 @pytest.mark.parametrize("num_stages", [2, 3])
 @pytest.mark.parametrize("num_warps", [4, 8])
 @pytest.mark.parametrize("use_fp8", [False, True])
+@pytest.mark.parametrize("flatten", [False, True])
```

Comment:
```suggestion
@pytest.mark.parametrize("flatten", [False, True] if is_blackwell() else [True])
```

This reduce the test matrix size on Hopper